typescript/src/lib/services/container_config.ts

```diff
@@ -12,110 +10,125 @@ export type FilesArtifactID = string;
 // ====================================================================================================
 // TODO defensive copy when we're giving back complex objects?????
 // Docs available at https://docs.kurtosistech.com/kurtosis-client/lib-documentation
-export class ContainerCreationConfig {
+export class ContainerConfig {
 	
     private readonly image: string;
-    private readonly kurtosisVolumeMountpoint: string; // Technically the enclave data volume, but we call it this for simplicity for the user
     private readonly usedPortsSet: Set<string>;
-    private readonly fileGeneratingFuncs: Map<string, (fp: number) => Promise<Result<null, Error>>>; // File descriptors are just integers
-    private readonly usedStaticFilesSet: Set<StaticFileID>;
+    private readonly kurtosisVolumeMountpoint: string; // Technically the enclave data volume, but we call it this for simplicity for the user
     private readonly filesArtifactMountpoints: Map<FilesArtifactID, string>;
+    private readonly entrypointOverrideArgs: string[];
+	private readonly cmdOverrideArgs: string[];
+	private readonly environmentVariableOverrides: Map<string,string>;
 
     constructor(
             image: string,
-            kurtosisVolumeMountpoint: string,
             usedPortsSet: Set<string>,
-            fileGeneratingFuncs: Map<string, (fp: number) => Promise<Result<null, Error>>>,
-            usedStaticFilesSet: Set<StaticFileID>,
-            filesArtifactMountpoints: Map<FilesArtifactID, string>){
+            kurtosisVolumeMountpoint: string,
+            filesArtifactMountpoints: Map<FilesArtifactID, string>,
+            entrypointOverrideArgs: string[],
+            cmdOverrideArgs: string[],
+            environmentVariableOverrides: Map<string,string>) {
         this.image = image;
-        this.kurtosisVolumeMountpoint = kurtosisVolumeMountpoint;
         this.usedPortsSet = usedPortsSet;
-        this.fileGeneratingFuncs = fileGeneratingFuncs;
-        this.usedStaticFilesSet = usedStaticFilesSet;
+        this.kurtosisVolumeMountpoint = kurtosisVolumeMountpoint;
```

Comment:
Thought (from what I sent you on Slack) - what if we just completely hardcode this, and don't make it configurable _anywhere_? I.e., just use the constant `DEFAULT_KURTOSIS_VOLUME_MOUNTPOINT` in `NetworkContext` everywhere